lib/domain/dtos/UpdateRunDto.js

```diff
@@ -39,6 +39,8 @@ const BodyDto = Joi.object({
     detectorsQualitiesChangeReason: Joi.string().optional(),
     calibrationStatus: Joi.string().valid(...RUN_CALIBRATION_STATUS).optional(),
     calibrationStatusChangeReason: Joi.string().optional(),
+    userIdO2Start: Joi.number().optional(),
+    userIdO2Stop: Joi.number().optional(),
```

Comment:
Don't bother with this API, you can leave it without it I suppose.

Reply:
done